docs/installation/introduction.rst

```diff
@@ -18,62 +17,61 @@ the following steps:
 #. :doc:`Post installation <./post-install>` configuration steps and sanity
    checks
 
+.. warning::
+
+   MetalK8s is not designed to handle world-distributed multi-site
+   architectures. Instead, it provides a highly resilient cluster at the
+   datacenter scale. To manage multiple sites, look into application-level
+   solutions or alternatives from such the communities as `the Multicluster SIG
+   <https://github.com/kubernetes/community/tree/master/sig-multicluster>`_).
+
 .. _Kubernetes: https://kubernetes.io/
 
 .. _installation-intro-architecture:
 
 Choosing a Deployment Architecture
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Before starting the installation, choosing an architecture is recommended, as
-it can impact sizing of the machines and other infrastructure-related details.
 
-.. note:: "Machines" may indicate bare-metal servers or VMs interchangeably.
-
-.. warning::
-
-   MetalK8s is not designed to handle world-distributed multi-sites
-   architectures. Instead, it focuses on providing a highly resilient cluster
-   at the datacenter scale. To manage multiple sites, look into solutions
-   provided at the application level, or alternatives from the community
-   (such as what `the SIG Multicluster
-   <https://github.com/kubernetes/community/tree/master/sig-multicluster>`_
-   provides).
+Before starting the installation, you must choose an architecture.
```

Comment:
The meaning changed. It was a recommendation, and became a requirement. The fact is, you don't **need** to have an architecture first - expanding, replacing servers, changing roles, is exactly where MetalK8s is much better than the RING.

Reply:
fixed.